pkg/arvo/sur/invite-store.hoon

```diff
@@ -0,0 +1,34 @@
+|%
+++  serial  @uvH
+::
++$  invite
+  $:  =path
+      =ship
+      recipient=ship
+      app=@tas
+      text=cord
+  ==
+::
++$  invitatory  (map serial invite)  ::  containing or conveying an invitation
+::
++$  invites  (map path invitatory)
+::
++$  invite-base
+  $%  [%create =path]
+      [%delete =path]
+      [%invite =path uid=serial =invite]
+      [%decline =path uid=serial]
+  ==
+::
++$  invite-action
+  $%  invite-base
+      [%accept =path uid=serial]
+  ==
+::
++$  invite-update
+  $%  invite-base
+      [%invitatory =invitatory]
+      [%accepted =path uid=serial =invite]
```

Comment:
Why do we re-send the invite here?

Reply:
Because accepting an invite deletes the invite, and apps may want to trigger some effect based off of the %accepted event. (As the %chat-hook does)

Reply:
Shouldn't it also be sent along in a `%decline` then?

Reply:
No, at least not until there's some use case. %declines don't cause an action.